contracts/voting-verifier/src/query.rs

```diff
@@ -31,12 +32,41 @@ pub fn is_verified(
         .collect::<Result<Vec<_>, _>>()
 }
 
-pub fn verification_status(
+pub fn is_worker_set_confirmed(deps: Deps, operators: &Operators) -> Result<bool, ContractError> {
```

Comment:
```suggestion
pub fn is_worker_set_verified(deps: Deps, operators: &Operators) -> Result<bool, ContractError> {
```
I dislike it that we mix the usage of verified and confirmed.